app/reducers/index.js

```diff
@@ -314,6 +318,14 @@ const redatApp = (state = defaultState, action) => {
           }
         }
       }
+    case 'TOGGLE_MENU':
```

Comment:
do you think it would make more sense to call it `TOGGLE_BUG_MENU` instead?

Reply:
Right now it only shows the debug information but I believe it was added to hold more information.

Reply:
aha i see. looks good,  i think we can merge it then